fcrepo-jms/src/test/java/org/fcrepo/integration/jms/observer/AbstractJmsIT.java

```diff
@@ -80,23 +87,29 @@
      */
     private static final long TIMEOUT = 20000;
 
-    private static final String testIngested = "/testMessageFromIngestion-" + randomUUID();
+    private String testIngested = "/testMessageFromIngestion-" + randomUUID();
```

Comment:
Is there a reason these are no longer static final?

Reply:
Yes, when they're static JUnit will not regenerate the values between test runs. The in memory containment index is not cleared between tests, as such the ids must be unique.